go.mod

```diff
@@ -28,8 +27,9 @@ require (
 	k8s.io/component-base v0.19.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.18.0
+	k8s.io/mount-utils v0.20.6
 	k8s.io/test-infra v0.0.0-20200115230622-70a5174aa78d
-	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
```

Comment:
This line should be removed.

Reply:
I see there are places like using k8s.io/utils/exec etc. 
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/search?q=%22k8s.io%2Futils%2Fexec%22

Reply:
So I think we should leave this line here? @msau42

Reply:
yup thanks for checking!